private/pkg/app/appcmd/appcmd.go

```diff
@@ -167,6 +170,40 @@ func run(
 	// commands.
 	cobraCommand.CompletionOptions.DisableDefaultCmd = true
 
+	// Add an explicit "completion" command to the root to cover the most
+	// common use case.
+	if command.IsRoot {
+		supportedShells := []string{"bash", "fish", "powershell", "zsh"}
```

Comment:
This variable should be removed if this is just for the error

Reply:
Done